packages/wonder-blocks-modal/components/two-column-modal.js

```diff
@@ -1,12 +1,59 @@
 // @flow
-import React from "react";
+/**
+ * A two-column modal layout.
+ */
+import * as React from "react";
 import {StyleSheet, css} from "aphrodite";
 
-export default class TwoColumnModal extends React.Component<{}> {
+import Color from "wonder-blocks-color";
+
+type Props = {
+    leftContent: React.Node,
+    rightContent: React.Node,
+};
+
+export default class TwoColumnModal extends React.Component<Props> {
     render() {
-        return <div>
-            Hello, world!
-            TODO(mdr): Build TwoColumnModal.
-        </div>;
+        return (
+            <div className={css(styles.container)}>
+                <div className={css(styles.column, styles.leftColumn)}>
+                    {this.props.leftContent}
+                </div>
+                <div className={css(styles.column, styles.rightColumn)}>
+                    {this.props.rightContent}
+                </div>
+            </div>
+        );
     }
 }
+
+const styles = StyleSheet.create({
+    container: {
+        display: "flex",
+        flexDirection: "row",
+        alignItems: "stretch",
+
+        // This minimum height is a fixed number from the spec.
+        minHeight: 464,
+    },
+
+    column: {
+        boxSizing: "border-box",
+        flex: "0 0 50%",
+        paddingTop: 64,
+        paddingBottom: 64,
```

Comment:
I don't think we should be building padding into the component since the top padding may be different depending if there's something above the title or not.

Reply:
Hmm right, makes sense. I imagine there's likely to be a concept of default padding that we want to ensure that call sites adhere to, but there will probably also be exceptions like the full-bleed case you mention. I'll check with Tabitha to see whether a default padding is part of her conception of the design—if so, maybe we can offer a flag to remove it?